.binstar.yml

```diff
@@ -4,18 +4,17 @@ platform:
   - linux-64
   - osx-64
   - win-64
-  - linux-32
-  - win-32
 
 engine:
   - python=2.7
   - python=3.4
   - python=3.5
 
-install:
-  - conda config --add channels anaconda-nb-extensions
-
 script:
-  - conda build --quiet conda.recipe --no-test
+  - >
+    conda build conda.recipe
+    -c bokeh
```

Comment:
we should not be using the bokeh channel, I can accept a general channel like `r` or `javascript` but if we need some package from the bokeh channel, we need to move it to our own anaconda-nb-extensions space...